packages/cozy-clisk/docs/RequestInterceptor.md

```diff
@@ -0,0 +1,77 @@
+# RequestInterceptor
+
+## Intercepting HTTP requests in clisk konnectors
+
+To intercept HTTP requests, you can init the RequestInterceptor like this :
+
+```javascript
+import {
+  ContentScript,
+  RequestInterceptor
+} from 'cozy-clisk/dist/contentscript'
+
+const requestInterceptor = new RequestInterceptor([
+  {
+    label: 'testRequest',
+    method: 'GET',
+    url: 'test/api',
+    serialization: 'json'
```

Comment:
What is it? Is it the `content/type` ? the accepted header?

Reply:
This is how we choose to serialize the body of the response, since it needs to be serialized to be transmitted from the worker to the pilot. We could relate it to the content type but I preferred it to be explicit.